### ISSUE TITLE
Comfee Board

### ISSUE THREAD

**Issue by fraschizzato** [Hypfer/esp8266-midea-dehumidifier#18]
Hi,
I have a Comfee CF-DEHU-12 without WiFi support. I've found that it has the internal connector but I can't get any data from UART (power is ok). I've checked the PCB routes and all seems connected (except from the lines that goes to the different display of other products, the "disp n5" on image). 
The fact is that I don't have the WiFi toggle button, can someone check in their unit what the WiFi button does? I suppose that only pulls up or down on pin of the IC.
Probably a wire check of the display connector to see which pins are shorted could be fine.
For any one interested the IC is a SINO WEALTH SH79F161AF (search for SH79F161A, the F is a package version). Used pins are TXD, RXD, P2.7 (generic PWM or INT) and a blank one (probably GND)
Thanks
![IMG_20220127_181919](https://user-images.githubusercontent.com/4145029/151435132-ecd82806-d673-4a96-a828-b1bf6a199b39.jpg)
![IMG_20220127_181931](https://user-images.githubusercontent.com/4145029/151435151-c12c3d7e-cbf0-48d7-acea-96b176bd2f59.jpg)

**Comment by puyoooooo** [comment]
> Hi, I have a Comfee MDDN-10DEN7 without WiFi support. I've found that it has the internal connector but I can't get any data from UART too power  15V. 
I don't have the WiFi toggle button too , 

![1](https://user-images.githubusercontent.com/95569918/213446799-ae4ad673-992b-4f80-8bb7-87f8a5d14e84.jpg)
![2](https://user-images.githubusercontent.com/95569918/213446809-ce3b76fc-ad1e-4a4d-b5b1-8fa70e32a354.jpg)
![3](https://user-images.githubusercontent.com/95569918/213446812-8f069329-6b59-4e67-9383-321e87b51ce3.jpg)
![4](https://user-images.githubusercontent.com/95569918/213446815-990bd100-faea-4bcc-b0d9-bd223d138faf.jpg)

**Comment by Meins321** [comment]
@fraschizzato did you ever figure it out?

**Comment by fraschizzato** [comment]
nope, as I remember my last check was about uart and it seems disabled by software

**Comment by Meins321** [comment]
ah thats more than unfortunate :-/
